example/.eslintrc

```diff
@@ -13,6 +13,9 @@
         "expect": true,
         "protractor": true
     },
+    "rules": {
+        "prefer-arrow-callback": 0
+    },
```

Comment:
That should only report if it finds an anonymous function. Named functions should be fine. My guess is it found an anon function it doesn't like.